### PR TITLE
Fix EXIT button position in creature window

### LINKED ISSUE #1200: "Exit" button in creature info window is shifted down
"Exit" button should be on the same level, as "Dismiss" button.
Upgrade creature

### PR DESCRIPTION
**Pull request by Moiman**
Fixes #1200

### CODE REVIEW

File: src/fheroes2/dialog/dialog_armyinfo.cpp [hunk 1]
```diff
@@ -92,7 +92,7 @@ int Dialog::ArmyInfo( const Troop & troop, int flags, bool isReflected )
 
     // button exit
     dst_pt.x = pos_rt.x + 415;
-    dst_pt.y = pos_rt.y + 225;
+    dst_pt.y = pos_rt.y + 222;
```
Comment (ihhub) [inline src/fheroes2/dialog/dialog_armyinfo.cpp:95]: Hi @Moiman , it works pretty well, unless you're in battle:
image

Could you please try this way: `dst_pt.y = pos_rt.y + ( ( UPGRADE & flags ) ? 222 : 225 );`

**Comment by ihhub** [comment]
@Moiman , thank you very much for your contribution to the project!

File: src/fheroes2/dialog/dialog_armyinfo.cpp (hunk 1 above)
Comment (Moiman) [reply src/fheroes2/dialog/dialog_armyinfo.cpp]: Maybe we could have moved dismiss button down to same level as exit button.

Comment (ihhub) [reply src/fheroes2/dialog/dialog_armyinfo.cpp]: The location is actually correcy for dismiss button so it's fine🙂

**Pull request merged**